tests/test_x509_ext.py

```diff
@@ -112,3 +129,94 @@ def test_unsupported_extension(self, backend):
         )
         extensions = cert.extensions
         assert len(extensions) == 0
+
+
+@pytest.mark.requires_backend_interface(interface=RSABackend)
+@pytest.mark.requires_backend_interface(interface=X509Backend)
+class TestRSABasicConstraintsExtension(object):
```

Comment:
Does this have anything to do with RSA, besides that some of the certs happen to be RSA?